vxquery-xtest/src/main/java/org/apache/vxquery/xtest/TestRunner.java

```diff
@@ -53,22 +59,61 @@
 
 public class TestRunner {
     private static final Pattern EMBEDDED_SYSERROR_PATTERN = Pattern.compile("(\\p{javaUpperCase}{4}\\d{4})");
-
+    private List<String> collectionList;
     private XTestOptions opts;
     private IHyracksClientConnection hcc;
     private IHyracksDataset hds;
 
     public TestRunner(XTestOptions opts) throws UnknownHostException {
         this.opts = opts;
+        this.collectionList = new ArrayList<String>();
     }
 
     public void open() throws Exception {
         hcc = TestClusterUtil.getConnection();
         hds = TestClusterUtil.getDataset();
     }
 
+    protected static XTestOptions getDefaultTestOptions() {
+        XTestOptions opts = new XTestOptions();
+        opts.verbose = false;
+        opts.threads = 1;
+        opts.showQuery = true;
+        opts.showResult = true;
+        opts.hdfsConf = "src/test/resources/hadoop/conf";
+        opts.catalog = StringUtils.join(new String[] { "src", "test", "resources", "VXQueryCatalog.xml" },
+                File.separator);
+        return opts;
+
+    }
+
     public TestCaseResult run(final TestCase testCase) {
         TestCaseResult res = new TestCaseResult(testCase);
+        TestConfiguration indexConf = new TestConfiguration();
+        indexConf.options = getDefaultTestOptions();
+        String baseDir = new File(getDefaultTestOptions().catalog).getParent();
+        try {
```

Comment:
These setting are set in the AbstractTestCaseFactory when loading a test.
For now, lets just include these options in getDefaultTestOptions(). Also rename getDefaultTestOptions to getShowIndexesTestOptions.